specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json

```diff
@@ -18,774 +18,827 @@
-                "readOnly": true,
-                "x-ms-enum": {
-                  "name": "AccountType",
-                  "modelAsString": true
-                }
-              },
-              "accountStatus": {
-                "description": "The current status of the billing account.",
-                "type": "string",
-                "enum": [
-                  "Active",
-                  "Deleted",
-                  "Disabled",
-                  "Expired",
-                  "Transferred",
-                  "Extended",
-                  "Terminated"
-                ],
-                "readOnly": true,
-                "x-ms-enum": {
-                  "name": "AccountStatus",
-                  "modelAsString": true
-                }
-              },
-              "billingProfiles": {
-                "description": "The billing profiles associated with the billing account. By default this is not populated, unless it's specified in $expand.",
-                "$ref": "#/definitions/BillingProfilesOnExpand"
-              },
-              "enrollmentDetails": {
-                "description": "The details about the associated legacy enrollment. By default this is not populated, unless it's specified in $expand.",
-                "$ref": "#/definitions/Enrollment",
-                "readOnly": true
-              },
-              "departments": {
-                "description": "The departments associated to the enrollment.",
-                "type": "array",
-                "items": {
-                  "$ref": "#/definitions/Department"
-                }
-              },
-              "enrollmentAccounts": {
-                "description": "The accounts associated to the enrollment.",
-                "type": "array",
-                "items": {
-                  "$ref": "#/definitions/EnrollmentAccount"
-                }
-              },
-              "hasReadAccess": {
-                "description": "Indicates whether user has read access to the billing account.",
-                "type": "boolean",
-                "readOnly": true
-              },
-              "tradeScreeningState": {
-                "description": "The result of a trade screening of customer.",
-                "type": "string",
-                "enum": [
-                  "Unknown",
-                  "NeedScreening",
-                  "UnderReview",
-                  "IngestionError",
-                  "Allowed",
-                  "NotAllowed"
-                ],
-                "readOnly": true,
-                "x-ms-enum": {
-                  "name": "tradeScreeningState",
-                  "modelAsString": true
+                "accountStatus": {
+                    "description": "The current status of the billing account.",
+                    "type": "string",
+                    "enum": [
+                        "Unknown",
+                        "Active",
+                        "Deleted",
```

Comment:
I thought we were going to remove "Deleted" and merge PendingReview/UnderReview.

Reply:
Correct for Deleted, I will remove it, for merging of two states PendingReview/UnderReview, still waiting on response from Trade team

Reply:
removed Deleted